app/controllers/items_controller.rb

```diff
@@ -18,17 +19,30 @@ def new
   def create
     item = Item.new(item_params)
     item.seller_id = current_user.id
-    @presenter = NewItemPresenter.new(item)
-    if @presenter.item.save
+    if item.valid?
+      item.save
+      if item.auction_end_time && item.starting_bid_price
```

Comment:
It might be nice to pull these two conditions into one method on item, like `item.auctioned?`.